UI/data/locale/en-US.ini

```diff
@@ -234,6 +234,14 @@ ConfirmStop.Text="Are you sure you want to stop the stream?"
 ConfirmStop.Text.Logout="You are about to logout - the stream will be stopped. Continue?"
 ConfirmStop.Text.ChangeRoom="You are changing the Evercast room - the stream will be stopped. Continue?"
 
+# evercast. re-login if room url is different
+Question.Title.RoomUrlDomainIsDifferent="Room URL domain is different"
+Question.Text.RoomUrlDomainIsDifferent="Room URL domain is different from the service you're logged-in. Re-login to a different service?"
```

Comment:
Suggest the following changes:
Message: "You are logged in to {domain.evercast.us}, but the room you have selected is in {domain2.evercast.us}.  Would you like to log in to {domain2.evercast.us}?"